ios/Classes/FacebookModule.m

```diff
@@ -279,10 +279,10 @@ -(void)setPermissions:(id)arg
  * facebook.logCustomEvent('clappedHands');
  *
  */
--(void)logCustomEvent:(id)args
+-(void)logCustomEvent:(id)event
 {
-    ENSURE_SINGLE_ARG(args, NSString);
-    NSString* event = [args objectAtIndex:0];
+    ENSURE_SINGLE_ARG(event, NSString);
+    //NSString* event = [args objectAtIndex:0];
     [FBSDKAppEvents logEvent:event];
```

Comment:
Use ´[FBSDKAppEvents logEvent:[TiUtils stringValue:event]];` instead and remove the old line instead of commenting it out.